vispy/scene/widgets/grid.py

```diff
@@ -346,7 +346,7 @@ def _add_widget_dim_constraints(solver, width_grid, height_grid,
 
         for ws in width_grid:
             for w in ws:
-                solver.addConstraint(w >= 0,)
```

Comment:
if you make this 1, and the one below as well, I think it might be fixed. Now in iter 148 without failing.

Reply:
844 iters ...